app/frontend/components/common/navbar/navbarAuth.js

```diff
@@ -51,11 +51,10 @@ const NavbarAuth = connect(
           onClick: () => window.history.pushState({}, 'txHistory', 'txHistory'),
         },
         h(AdaLiteLogo),
-        h('span', undefined, 'AdaLite'),
-
-        h('sup', undefined, '⍺')
+        h('span', undefined, 'AdaLite')
       ),
       isDemoWallet ? h('div', {class: 'public-wallet-badge pulse'}, 'DEMO WALLET') : null,
+      h('div', {class: 'logo-version-number'}, '1.0.0'),
```

Comment:
I would fetch the app version from package.json. I suggest using the one in the root directory. I would remove the "version" field from the second package.json (the one in the "app" directory) to avoid confusion.

The helper to get the app version could be placed in app/helpers directory and could look something like this (tested and working):

```
const {version: appVersion} = require('../../../package.json')

export default appVersion
```

I'm aware that this approach would not work for the landing page, but I see no point in displaying the version there, so I would simply remove it from there